### ISSUE TITLE
预测的PR值均为0，请问是为什么

### ISSUE THREAD

**Issue by zhangkun9613** [macanv/BERT-BiLSTM-CRF-NER#146]
I:NER Training:[ber:tra:647]:***** Running prediction*****
I:NER Training:[ber:tra:648]:  Num examples = 2923
I:NER Training:[ber:tra:649]:  Batch size = 64
I:NER Training:[ber:mod:370]:*** Features ***
I:NER Training:[ber:mod:372]:  name = input_ids, shape = (?, 40)
I:NER Training:[ber:mod:372]:  name = input_mask, shape = (?, 40)
I:NER Training:[ber:mod:372]:  name = label_ids, shape = (?, 40)
I:NER Training:[ber:mod:372]:  name = segment_ids, shape = (?, 40)
shape of input_ids (?, 40)
2019-06-01 17:17:23.984091: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1511] Adding visible gpu devices: 0
2019-06-01 17:17:23.984159: I tensorflow/core/common_runtime/gpu/gpu_device.cc:982] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-06-01 17:17:23.984180: I tensorflow/core/common_runtime/gpu/gpu_device.cc:988]      0 
2019-06-01 17:17:23.984191: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1001] 0:   N 
2019-06-01 17:17:23.984376: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 21553 MB memory) -> physical GPU (device: 0, name: Tesla P40, pci bus id: 0000:02:00.0, compute capability: 6.1)
processed 30086 tokens with 0 phrases; found: 0 phrases; correct: 0.
accuracy:  96.68%; precision:   0.00%; recall:   0.00%; FB1:   0.00

**Comment by nuass** [comment]
请问这个问题解决了吗？我也有同样的问题，
processed 68872 tokens with 874 phrases; found: 0 phrases; correct: 0.
accuracy:  79.65%; precision:   0.00%; recall:   0.00%; FB1:   0.00
              ORG: precision:   0.00%; recall:   0.00%; FB1:   0.00  0
